tests/api/test_v1_push.py

```diff
@@ -9,6 +10,12 @@
 @pytest.mark.django_db
 class TestV1PushProxy:
 
+    @pytest.fixture(autouse=True)
+    def mock_disambiguate(self):
```

Comment:
Ohhh, brain fart. I even double-checked the behavior of `autouse` but apparently do not know how to read.